tools/downloader/README.md

```diff
@@ -286,15 +290,109 @@ executed commands, or "auto", in which case the number of CPUs in the system is
 By default, all commands are run sequentially.
 
 The script can print the conversion commands without actually running them.
-To do this, use the `--dry-run` option:
+To do this, use the `--dry_run` option:
+
+```sh
+./converter.py --all --dry_run
+```
+
+See the "Shared options" section for information on other options accepted by
+the script.
+
+Model quantizer usage
+---------------------
+
+Before you run the model quantizer, you must prepare a directory with
+the datasets required for the quantization process. This directory will be
+referred to as `<DATASET_DIR>` below. See the "Dataset directory layout"
+section for information on the expected contents of that directory.
+
+The basic usage is to run the script like this:
+
+```sh
+./quantizer.py --all --dataset_dir <DATASET_DIR>
+```
+
+This will quantize all models for which quantization is supported. Other models
+are ignored.
```

Comment:
Question: where (how) can I find list of supported model?

Reply:
You could use `--all --dry_run` to see which models will get quantized. I'm considering adding a field to the info dumper's output to indicate which precisions each model can be quantized to, but I haven't done so yet, because it seems of secondary importance and there's not much time before the release.

Reply:
Should we add something to documentation?
E.g.:
| Metric            | Value         |
|-------------------|---------------|
| Type              | Detection     |
| GFLOPs            | 5.424         |
| MParams           | 11.229        |
| **Quantazible** | **Yes** |
| Source framework  | TensorFlow.js\*  |

@vladimir-dudnik what do you think?

Reply:
I'd prefer to avoid manual maintenance of this info in different places.

Reply:
I'd prefer to avoid manual maintenance of this info in different places